### DISCUSSION TITLE
can we control/drive query response structure using a variable

### DISCUSSION THREAD

**Discussion by vishalbadehotmail** [graphql/graphql-spec#1051] [Q&A]
Hi,
In a GraphQL query we specify exactly what fields/structure we need in the output/response.

can this be dynamic, like a variable/fragment holds this data and will be used instead of hardcoding the fields?

Thank you.

**Comment by benjie** [comment]
No; the closest you have to that are the `@skip` and `@include` directives that allow you to turn certain parts of the operation off/on based on variable inputs.
